<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TravelAgencyUI</class>
 <widget class="QMainWindow" name="TravelAgencyUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1168</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TravelAgencyUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Reise ID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="reiseID"/>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Kunde:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="nameLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>ID:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="idLineEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Reisen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <column>
           <property name="text">
            <string>Reise</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Beginn der Reise</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Ende der Reise</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Buchungen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget_2">
        <column>
         <property name="text">
          <string>Buchungstyp</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Beginn der Reise</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Ende der Reise</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Preis</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1168</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionEinlesen"/>
   <addaction name="separator"/>
   <addaction name="actionSuchen"/>
   <addaction name="actionSpeichern"/>
   <addaction name="actionSettings"/>
   <addaction name="actionErrors"/>
  </widget>
  <action name="actionEinlesen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/speichern.png</normaloff>:/speichern.png</iconset>
   </property>
   <property name="text">
    <string>Einlesen</string>
   </property>
  </action>
  <action name="actionSuchen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/search.png</normaloff>:/search.png</iconset>
   </property>
   <property name="text">
    <string>Suchen</string>
   </property>
  </action>
  <action name="actionSpeichern">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/speichern.png</normaloff>
     <normalon>:/speichern.png</normalon>:/speichern.png</iconset>
   </property>
   <property name="text">
    <string>Speichern</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/settings.png</normaloff>
     <normalon>:/settings.png</normalon>:/settings.png</iconset>
   </property>
   <property name="text">
    <string>actionSettings</string>
   </property>
  </action>
  <action name="actionErrors">
   <property name="text">
    <string>Errors</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
